bundles/org.openhab.binding.http/README.md

```diff
@@ -71,13 +71,15 @@ The relevant transformation service needs to be installed via the Main UI or add
 
 Here are a few examples to unwrap an incoming value via `stateTransformation` from a complex response:
 
-| Received value                                                      | Tr. Service | Transformation                            |
-|---------------------------------------------------------------------|-------------|-------------------------------------------|
-| `{device: {status: { temperature: 23.2 }}}`                         | JSONPATH    | `JSONPATH:$.device.status.temperature`    |
-| `<device><status><temperature>23.2</temperature></status></device>` | XPath       | `XPath:/device/status/temperature/text()` |
-| `THEVALUE:23.2°C`                                                   | REGEX       | `REGEX::(.*?)°`                           |
+| Received value                                                      | Tr. Service | Transformation                             |
+|---------------------------------------------------------------------|-------------|--------------------------------------------|
+| `{device: {status: { temperature: 23.2 }}}`                         | JSONPATH    | `JSONPATH($.device.status.temperature)`    |
+| `<device><status><temperature>23.2</temperature></status></device>` | XPath       | `XPath(/device/status/temperature/text())` |
+| `THEVALUE:23.2°C`                                                   | REGEX       | `REGEX(:(.*?)°)`                           |
 
-Transformations can be chained by separating them with the mathematical intersection character "∩".
+Transformations can be chained in the UI by listing each transformation on a separate line, or by separating them with the mathematical intersection character "∩".
+Starting from openHAB 4.3, transformations are defined using this syntax: `TYPE(FUNCTION)`, e.g.: `JSONPATH($.path)`.
```

Comment:
I don‘t think we should mention 4.3 here — these doc changes will only be on next.openhab.org until the 4.3 release.

Reply:
What should it say instead?

Reply:
I would go for:

> Transformations are defined using this syntax: …
> The syntax `TYPE:FUNCTION` is still supported, e.g. …

Reply:
Thanks, updated!